setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="starlette_exporter",
-    version="0.15.1",
+    version="0.15.2",
```

Comment:
Going to bump the minor version to `0.16.0` to indicate there's a new feature here.  I'll take care of it when I create the release though, just letting you know.